temba/channels/models.py

```diff
@@ -211,6 +211,7 @@ class Channel(TembaModel):
     CONFIG_MAX_LENGTH = 'max_length'
     CONFIG_MACROKIOSK_SENDER_ID = 'macrokiosk_sender_id'
     CONFIG_MACROKIOSK_SERVICE_ID = 'macrokiosk_service_id'
+    CONFIG_RP_HOSTNAME_OVERRIDE = 'rp_hostname_override'
```

Comment:
@nicpottier In the Uganda data centre the routing to the hostname we have for RapidPro would go via the internet and because their bandwidth is so terrible we had lots of timeouts, so we needed to override the hostname to use an internal address